addons/website_blog/static/src/scss/website_blog.scss

```diff
@@ -305,3 +305,7 @@ $o-wblog-loader-size: 50px;
         background-color: rgba(black, 0.7);
     }
 }
+
+#o_wblog_index_content > .container > .row > .border-end {
+    width: auto;
+}
```

Comment:
- There has to be a comment here to explain why this very specific rule is necessary
- There should also be a "TODO adapt in master: ..." comment as we'll obviously not keep such a rule over there :wink: 
- While the CSS fix is nice in stable, I wonder why the actual fix in master will be: won't it be about adding a class in the XML? Maybe it can already be done in 16.0 as well if this is what we need ? (but I don't know: it should be explained in a comment here).

Reply:
> I wonder why the actual fix in master will be: won't it be about adding a class in the XML? 

Yes, that is what I thought for master as well: maybe something like this in `website_blog_templates.xml`:
```diff
-        <div t-if="opt_blog_list_view" class="border-end d-none d-lg-block" style="opacity: 0.5"/>
-        <div t-attf-class="col-12 col-md-3 d-flex #{opt_blog_list_view and 'col-lg-4' or 'ms-lg-5'}">
+        <div t-attf-class="col-12 col-md-3 d-flex #{opt_blog_list_view and 'col-lg-4' or 'ms-lg-5'} #{'border-start' if opt_blog_list_view else ''}"
```

Reply:
@qsm-odoo , do you have a preference regarding this last three propositions for master? :relaxed:

Reply:
@loco-odoo Probably the 4th option: adding the w-auto class on the element :wink: If we want to do solution 1 (and thus change the design right?), I guess it requires a validation from PDE

Reply:
@qsm-odoo , thanks to PDE, this border element should be removed in master :+1: